solidity-plugin/src/main/java/org/sonarsource/solidity/SoliditySensor.java

```diff
@@ -100,12 +99,13 @@ public void analyzeFiles(SensorContext context, List<InputFile> files) {
 
   private FileMeasures computeMeasures(SolidityParser parser, FileLinesContext fileLinesContext, InputFile file) {
     MetricsVisitor metricsVisitor = new MetricsVisitor(parser);
-    try {
-      CognitiveComplexityVisitor cognComplex = new CognitiveComplexityVisitor(Utils.returnParserUnitFromParsedFile(file.contents()).sourceUnit());
-
-    } catch (RecognitionException | IOException e) {
-      LOG.debug(e.getMessage(), e);
-    }
+    // try {
+    // CognitiveComplexityVisitor cognComplex = new
+    // CognitiveComplexityVisitor(Utils.returnParserUnitFromParsedFile(file.contents()).sourceUnit());
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=20&issues=AWQdvLOs_bEzQ-kUUpnG&open=AWQdvLOs_bEzQ-kUUpnG)

[ISSUE_KEY]: AWQdvLOs_bEzQ-kUUpnG